css/main.css

```diff
@@ -411,3 +455,31 @@ footer {
     font-size: 18px;
   }
 }
+
+@media screen and (min-width: 500px) {
```

Comment:
Great that you took time and effort to understand mobile responsiveness 🎉

Reply:
Thank you so much☺️